guide/C/ch_txns.xml

```diff
@@ -349,10 +349,10 @@
         <note>
           <para>Note: An account register displays a transaction when that
           transaction has a split assigned to the account. If a given transaction
-          has more than one split assigned to a single account, then in Basic and
-          Auto-Split view modes that transaction will appear in the register one
+          has more than one split assigned to the register account, then in Basic and
```

Comment:
The text, before I modified it, says to me that a transaction is shown whenever there is more than 1 split to "any" single account but it is only when that account is the account that was "opened" (or Open SubAccounts) that this occurs.
How about:
An account register displays a transaction when that transaction has a split assigned to the
account. If a given transaction has more than one split assigned to the account, then in Basic
and Auto-Split view that transaction will appear in the register one time for each split
assigned to that account.